apiserver/DEVELOPMENT.md

```diff
@@ -97,36 +97,7 @@ To update the swagger ui files deployed with the Kuberay API server, you'll need
 * Manually run the [hack/update-swagger-ui.bash](hack/update-swagger-ui.bash) script. The script downloads the swagger ui release and copies the downloaded files
 to the [../third_party/swagger-ui](../third_party/swagger-ui/) directory. It copies the [swagger-initializer.js](../third_party/swagger-ui/swagger-initializer.js)
 to [swagger-initializer.js.backup](../third_party/swagger-ui/swagger-initializer.js.backup).
-* Update the contents of the [swagger-initializer.js](../third_party/swagger-ui/swagger-initializer.js) to set the URLs for for the individual swagger docs. The content of the file is show below:
-
-    ```javascript
-    window.onload = function() {
-    //<editor-fold desc="Changeable Configuration Block">
-
-    // the following lines will be replaced by docker/configurator, when it runs in a docker-container
-    window.ui = SwaggerUIBundle({
-        spec: location.host,
-        urls:  [{"url":"http://"+location.host+"/swagger/serve.swagger.json","name":"RayServe Service"},
-                {"url":"http://"+location.host+"/swagger/error.swagger.json","name":"Errors API"},
-                {"url":"http://"+location.host+"/swagger/job.swagger.json","name":"RayJob Service"},
-                {"url":"http://"+location.host+"/swagger/config.swagger.json","name":"ComputeTemplate Service"},
-                {"url":"http://"+location.host+"/swagger/cluster.swagger.json","name":"Cluster Service"}],
-        dom_id: '#swagger-ui',
-        deepLinking: true,
-        presets: [
-        SwaggerUIBundle.presets.apis,
-        SwaggerUIStandalonePreset
-        ],
-        plugins: [
-        SwaggerUIBundle.plugins.DownloadUrl
-        ],
-        layout: "StandaloneLayout"
-    });
-
-    //</editor-fold>
-    };
-    ```
```

Comment:
I don't think this should be inlined, since the actual file is already hyperlinked. It just means one more thing to update.